### PR TITLE
[ADD] open_academy: New module to manage courses T#59013

### PR DESCRIPTION
**Pull request by geraspace**
- Add new model "open.academy.course".
- Add demo data for open.academy.course model.
- Fix some lints.

### CODE REVIEW

**Comment by geraspace** [comment]
@edy1192

File: open_academy/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Open Academy",
+
```
Comment (edy1192) [inline open_academy/__manifest__.py]: Please remove all blank lines between keys.

```suggestion
'name': "Open Academy",
'summary': """
    Module description""",
```

File: open_academy/controllers/controllers.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# from odoo import http
+
```
Comment (edy1192) [inline open_academy/controllers/controllers.py]: Please don't add commented code.  Remove this file and all the references to it. When you need this you'll add it.

File: open_academy/demo/demo.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<odoo>
+    <data>
+<!--
```
Comment (edy1192) [inline open_academy/demo/demo.xml]: Same here. Don't add commented code. Please remove this file and all the references to it.

File: open_academy/demo/open_academy_demo.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <record id="open_academy_course_nikon" model="open.academy.course">
+      <field name="name">Curso Nikon</field>
```
Comment (edy1192) [inline open_academy/demo/open_academy_demo.xml]: All the code must be in English please.

```suggestion
      <field name="name">Nikon Course</field>
```

File: open_academy/demo/open_academy_demo.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      <field name="description">A cool Nikon course</field>
+    </record>
+    <record id="open_academy_course_canon" model="open.academy.course">
+      <field name="name">Curso Canon</field>
```
Comment (edy1192) [inline open_academy/demo/open_academy_demo.xml]: Same here, English please.

File: open_academy/demo/open_academy_demo.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+      <field name="description">A cool Canon course</field>
+    </record>
+    <record id="open_academy_course_leica" model="open.academy.course">
+      <field name="name">Curso Leica</field>
```
Comment (edy1192) [inline open_academy/demo/open_academy_demo.xml]: Same here, English please.

File: open_academy/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1 @@
+from . import open_academy
```
Comment (edy1192) [inline open_academy/models/__init__.py]: Please name the file like the model. 

```suggestion
from . import open_academy_course
```
It is a standard that allows us to know which model is being added or modified, just by looking at the file name.

File: open_academy/models/open_academy.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (edy1192) [inline open_academy/models/open_academy.py]: Remember change the file name please.

File: open_academy/views/templates.xml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <data>
+<!--
```
Comment (edy1192) [inline open_academy/views/templates.xml]: Same here. Don't add commented code. Please remove this file and all the references to it.

File: open_academy/views/views.xml [hunk 10]
```diff
@@ -0,0 +1,61 @@
+<odoo>
+  <data>
+    <!-- explicit list view definition -->
+<!--
```
Comment (edy1192) [inline open_academy/views/views.xml]: Same here. Don't add commented code. Please remove this file and all the references to it.

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by edy1192** [comment]
Hi @geraspace revision done. 

If you have any doubts you could ask me. 

Regards.

File: open_academy/__manifest__.py [hunk 11]
```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Open Academy",
```
Comment (edy1192) [inline open_academy/__manifest__.py]: Please keep consistent if you will use single quote do it everywhere, same for double quotes, a good practice it's use double quotes.

File: open_academy/__manifest__.py [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Test module',
+    'license': 'LGPL-3',
+    'version': "15.0.1.0.1",
+
+    # any module necessary for this one to work correctly
+    'depends': ['base'],
```
Comment (edy1192) [inline open_academy/__manifest__.py]:
```suggestion
    'depends': [
    'base',
    ],
```

File: open_academy/__init__.py [hunk 13]
```diff
@@ -0,0 +1,3 @@
+from . import controllers
```
Comment (edy1192) [inline open_academy/__init__.py]: Controllers are not needed at this point, so remove the import and also the folder

File: open_academy/__init__.py [hunk 14]
```diff
@@ -0,0 +1,3 @@
...
+from . import models
+from . import demo
```
Comment (edy1192) [inline open_academy/__init__.py]: There's no a "demo.py" file, this is for import models (python's files)

**Review by edy1192 — CHANGES_REQUESTED** [review]

File: open_academy/demo/open_academy_demo.xml [hunk 15]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (edy1192) [inline open_academy/demo/open_academy_demo.xml]: Please rename this file `open_academy_course_demo.xml`

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by geraspace** [comment]
Hi @edy1192, could you check the update please? And thank you!

File: open_academy/__manifest__.py (hunk 11 above)
Comment (edy1192) [reply open_academy/__manifest__.py]: This change is pending. Please use double quotes in all keys

```suggestion
    "name": "Open Academy",
```

File: open_academy/demo/demo.xml (hunk 3 above)
Comment (edy1192) [reply open_academy/demo/demo.xml]: This change is pending. Remove this file please and its reference in `__manifest__.py` file.

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by edy1192** [comment]
Hi @geraspace There are 2 pending comments, please apply them and let me know again to review.

Regards.

**Comment by geraspace** [comment]
@edy1192 I think it's ready now.

**Review by edy1192 — APPROVED** [review]

**Comment by edy1192** [comment]
Thank you @geraspace

**Pull request merged**